public/v2/apps/etherpad.json

```diff
@@ -0,0 +1,77 @@
+{
+    "captainVersion": "2",
+    "documentation": "https://github.com/ether/etherpad-lite/blob/develop/doc/docker.md ",
+    "displayName": "Etherpad",
+    "description": "Etherpad is a real-time collaborative editor for the web",
+    "dockerCompose": {
+        "version": "3.8",
+        "services": {
+            "$$cap_appname": {
+            "image": "etherpad/etherpad:$$cap_etherpad_version",
+            "ports": [
+                "9001:9001"
+            ],
+            "containerHttpPort": "9001",
+            "depends_on": [
+                "$$cap_appname-db"
+            ],
+            "environment": {
+                "NODE_ENV": "$$cap_env",
+                "DB_TYPE": "mysql",
+                "DB_HOST": "srv-captain--$$cap_appname-db",
+                "DB_PORT": "3306",
+                "DB_NAME": "etherpad",
+                "DB_USER": "etherpad",
+                "DB_PASS": "$$cap_db_pass",
+                "DB_CHARSET": "utf8mb4",
+                "TRUST_PROXY": "true"
+             }

```

Comment:
There is another env var in their docs, `ADMIN_PASSWORD`, do we not need it?
I am not an etherpad user, not sure if it's needed..

Reply:
If there is no admin_password set, the default is to not create an admin user.
With the choosen setup there is no authentification, so an admin user isn't extra useful
It's one less moving part